premium/usage.go

```diff
@@ -730,7 +730,7 @@ func (u *BatchUpdater) getTeamNameByTokenType(tokenType auth.TokenType) (string,
 
 func determineInstallationID(tokenType auth.TokenType) string {
 	switch tokenType {
-	case auth.SyncRunAPIKey, auth.SyncTestConnectionAPIKey:
+	case auth.APIKey:
 		return os.Getenv("_CQ_INSTALLATION_ID")
```

Comment:
Why switch on token type at all and not always return `os.Getenv("_CQ_INSTALLATION_ID")`?

Reply:
We can also do that, yes

Reply:
Just did that in https://github.com/cloudquery/plugin-sdk/pull/2140/commits/63fb2dcfa2a18c09b891c37442c9d3a588762200